test/getGrist.ts

```diff
@@ -23,6 +23,13 @@ export function getGrist(): GristUtils {
     // TODO: mocha-webdriver has a way of explicitly connecting a
     // server that might have advantages for debugging.
     await grist.wait();
+    // "Changes you made may not be saved" alerts are consumed
+    // in grist-core tests - do the same here.
+    // TODO: figure out why they occur.
```

Comment:
If it helps: they now appear in Grist when navigating away from the page while the current document is unsaved. (Many tests create a temporary, unsaved document.)

This was added recently.